caffe2/core/context_gpu.cu

```diff
@@ -268,8 +274,13 @@ CUDAContext::CUDAContext(const DeviceOption& option)
   DCHECK_EQ(option.device_type(), CUDA);
 }
 
+std::mutex& CUDAContext::mutex() {
+  return gCUDAContextMutex();
```

Comment:
Out of curiosity, can we save 3 lines of code by putting gCUDAContextMutex() into this function?